data/campaigns/atl02.wmf/elemental

```diff
@@ -0,0 +1,13 @@
+# Automatically created by Widelands 1.0~git24923[de6c28e@static_link] (Release)
+
+[global]
+packet_version="1"
+map_w="176"
+map_h="176"
+nr_players="3"
+name="Struggling for a new home again"
+author="SirVer,Nasenbaer,hessenfarmer"
+descr="After the quick escape from the drowning island, the atlanteans around Jundlina are faced with new challenges."
```

Comment:
Name and Descr need i18n markup. Also the Name should be Title Case.
```suggestion
name=_"Struggling For a New Home Again"
author="SirVer,Nasenbaer,hessenfarmer"
descr=_"After the quick escape from the drowning island, the Atlanteans around Jundlina are faced with new challenges."
```
The description doesn't really tell the player anything. That's not a problem, but perhaps you'd like to give a small hint at what sort of challenges they might encounter?